packages/nimble-components/src/wafer-map/modules/wafer-map-validator.ts

```diff
@@ -40,4 +43,48 @@ export class WaferMapValidator {
         }
         return !this.invalidGridDimensions;
     }
+
+    public validateDiesTableSchema(): boolean {
```

Comment:
Can probably save off the indexes to reduce the verbosity in this method to `colIndexField`, `rowIndexField`, `valueField`, etc. Seems unnecessary to have to redo the linear searches to find the indexes again.